src/d3a/models/market/grid_fees/base_model.py

```diff
@@ -0,0 +1,92 @@
+from d3a.models.market.grid_fees import GridFees
+
+
+class BaseModel(GridFees):
```

Comment:
IMO BaseModel is a bid generic. When calling one of the function one could easily get confused. How about renaming it to e.g. GridFees and rename the Baseclass in __init__.py to BaseClassGridFees ?

Reply:
Agreed, fixing that. Thanks!

Reply:
Just for my better understanding would you re-write all of GridFees class method for PACGridFee calculation?

Reply:
I do not plan to actually. PAC has mostly been dealt with in the context of this story, but not completely yet (it will be done in the context of [this story](https://gridsingularity.atlassian.net/browse/D3ASIM-1449) . However, from a quick look on Dirk's documentation, the fee calculation on the PAC market is almost identical to PAB. Therefore I do not think that this will be needed. Despite that, if the needed calculation is entirely different, then I will rewrite or override the class methods for PAC, yes. 
The main rational behind making this an interface though, is for creating completely different grid fee models. For instance, if a customer wants something like that, we can write another class that supports a combination of constant fees and ratios, with minimal changes on IAA or market class code.